.github/workflows/ci.yml

```diff
@@ -93,18 +93,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [
+        build:
+          [
             linux-x64-gnu,
             linux-x64-musl,
             linux-arm64-gnu,
             linux-arm64-musl,
             linux-ia32-gnu,
             darwin-arm64,
             darwin-x64,
-            # win32-x64-msvc,
-            # win32-x64-gnu,
-            # win32-arm64-msvc,
-            # win32-ia32-gnu,
+            win32-x64-msvc,
+            win32-x64-gnu,
+            win32-arm64-msvc,
+            win32-ia32-gnu,
           ]
```

Comment:
Am already fixing this in https://github.com/tailcallhq/tailcall/pull/1460/files